test/tornado_tests/test_motor_auth.py

```diff
@@ -70,7 +70,7 @@ async def test_scram(self):
                     await client.scramtestdb.collection.insert_one({})
 
         # No mechanism specified, always works.
-        for user, mechanism, should_work in [
+        for user, _mechanism, _should_work in [
```

Comment:
Is this a bug? _mechanism and _should_work appear unused

Reply:
adding an underscore indicates to flake that they are not used, I'll just make them regular underscores.

Reply:
Yeah I get that but I'm wondering why they are unused. Can we just simplify this to:
```
for user in ("sha1", "sha256", "both"):...
```

Reply:
Ah, right, done